spacy/ml/models/textcat.py

```diff
@@ -130,7 +130,7 @@ def build_text_classifier_v2(
         model = (linear_model | cnn_model) >> output_layer
         model.set_ref("tok2vec", tok2vec)
     if model.has_dim("nO") is not False:
-        model.set_dim("nO", nO)  # type: ignore[arg-type]
+        model.set_dim("nO", cast(int, nO))
```

Comment:
See other comment about `nO`.

Reply:
I've made a corresponding change in the branch, but for some reason it isn't being picked up here.

Reply:
Some caching invalidation issue in GitHub? I'll resolve this :).